mscore/exportxml.cpp

```diff
@@ -1156,12 +1156,12 @@ void ExportMusicXml::credits(Xml& xml)
       // determine page formatting
       const PageFormat* pf = _score->pageFormat();
       if (!pf) return;
-      const double h  = getTenthsFromInches(pf->size().height());
-      const double w  = getTenthsFromInches(pf->size().width());
-      const double lm = getTenthsFromInches(pf->oddLeftMargin());
-      const double rm = getTenthsFromInches(pf->oddRightMargin());
-      //const double tm = getTenthsFromInches(pf->oddTopMargin());
-      const double bm = getTenthsFromInches(pf->oddBottomMargin());
+      const double h  = pf->size().height()   * PageFormat::SCALE_XML;
```

Comment:
They are at 1/10th of a space.  Staff space == 5 pt/px.  1/10th staff space = 0.5pt/px.  PtPx = 72dpi.  SCALE_XML = 144dpi.
Is this possibly a problem is Staff space != 5pt/px?  Maybe that is a real problem.  I need to calculate SCALE_XML as a variable instead of making it a constant?  Or is this supposed to be relative to Staff Space?